radlab-ui/webapp/__mocks__/terraform.ts

```diff
@@ -186,7 +192,7 @@ variable "billing_budget_alert_spent_percents" {
 }
 
 variable "billing_budget_services" {
-  description = "A list of services ids to be included in the budget. If omitted, all services will be included in the budget. Service ids can be found at https://cloud.google.com/skus/. {{UIMeta group=0 order=12 updatesafe }}"
+  description = "A list of services ids to be included in the budget. If omitted, all services will be included in the budget. Service ids can be found at https://cloud.google.com/skus/. {{UIMeta group=0 order=12 updatesafe dependson=((enable_gpu_driver==true||set_external_ip_policy==true)&&(create_network==true||create_usermanaged_notebook==true))}}"
```

Comment:
Please remove the **first** and **last** parentheses wrapping the `dependsons` content